### PR TITLE
minor: Update dependency minor versions

### PR DESCRIPTION
**Pull request by lf-**
bors r+
🤖

### CODE REVIEW

**Comment by lf-** [comment]
damn, my git-sin script had a bug in it 🤣 
bors r+

File: Cargo.lock [hunk 1]
```diff
@@ -1943,3 +1943,15 @@ dependencies = [
  "xflags",
  "xshell",
 ]
+
```
Comment (lnicola) [inline Cargo.lock:1946]: Are these intentional?

Comment (lf-) [reply Cargo.lock:1946]: cargo-update generated them.

Comment (lf-) [reply Cargo.lock:1946]: OOps

Comment (lf-) [reply Cargo.lock:1946]: oops actually lol

**Comment by lnicola** [comment]
bors r-

> cargo-update generated them.

Ugh, that's strange. Do you have any local `Cargo.toml` changes?

And in the meanwhile, can you pin `libc` to 0.2.99? I wanted to open a PR, but then I saw yours.

**Comment by lf-** [comment]
> bors r-
> 
> > cargo-update generated them.
> 
> Ugh, that's strange. Do you have any local `Cargo.toml` changes?
> 
> And in the meanwhile, can you pin `libc` to 0.2.99 in `crates/rust-analyzer/Cargo.toml` (I hope that works)? I wanted to open a PR, but then I saw yours. It's needed because newer versions don't build on FreeBSD.

Done.

**Comment by matklad** [comment]
FWIW, I've extended my gbors script to give an ooportunity to prep a PR description out of concatenated commit messages: https://github.com/matklad/config/blob/8ccb77cba70ad052ce121c4cfd9a57c8ee39dd93/tool/src/gbors.rs

**Comment by lf-** [comment]
> FWIW, I've extended my gbors script to give an ooportunity to prep a PR description out of concatenated commit messages: https://github.com/matklad/config/blob/8ccb77cba70ad052ce121c4cfd9a57c8ee39dd93/tool/src/gbors.rs

Ah yes, so I can own myself harder and faster 🤣

**Comment by matklad** [comment]
Yeah; julia has known startup latency problems (known as TTFP -- time to first plot), and, after careful benchmarking and taking Amdahl's law into consideration, I came to the conclusion that `gbors` performance is the main limiting factor for my productivity, hence the Rust rewrite.

**Comment by bjorn3** [comment]
Why not make gbors a self-compiling rust script? :) https://github.com/bjorn3/rustc_codegen_cranelift/blob/master/y.rs#L1-L8

**Comment by matklad** [comment]
That's lovely! I do want to use dependencies though (xshell and anyhow), and I want to have a busy-box style single binary which dispatches based on argv[0]. 

Also, I wonder why we don't have an RFC for `rustc run main.rs`? That feels like a feature we should have...

**Comment by matklad** [comment]
(btw, the stuff is sort-of self-linking at least: https://github.com/matklad/config/blob/7371bb2e043ce37be872e25f0871baacba033747/tool/src/main.rs#L30)

**Pull request merged**